fdbserver/BlobWorker.actor.cpp

```diff
@@ -438,6 +438,10 @@ void checkGranuleLock(int64_t epoch, int64_t seqno, int64_t ownerEpoch, int64_t
 		throw granule_assignment_conflict();
 	}
 }
+
+bool isEncryptionOpSupported(EncryptionAtRestMode encryptMode) {
+	return encryptMode.isEncryptionEnabled() && SERVER_KNOBS->BG_METADATA_SOURCE == "tenant";
```

Comment:
For better understanding: `SERVER_KNOBS->BG_METADATA_SOURCE` does this correspond to Cluster `TenantMode` support? If yes, then shouldn't this be taken care as part of database config checks? 

If not, shouldn't we need to fix it as well? Seems scary otherwise.

Reply:
Not sure i'm wondering if @sfc-gh-jslocum would know?